src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.DefaultProxyCredentials.cs

```diff
@@ -90,7 +90,7 @@ public async Task ProxyExplicitlyProvided_DefaultCredentials_Ignored()
         public async Task ProxySetViaEnvironmentVariable_DefaultProxyCredentialsUsed(bool useProxy)
         {
             const string ExpectedUsername = "rightusername";
-            const string ExpectedPassword = "rightpassword";
+            string ExpectedPassword = SecretsHelper.SafePassword("rightpassword");
```

Comment:
The purpose of this method is to annotate that the password is for test purpose and can be part of source code.